config-sample.yaml

```diff
@@ -107,6 +107,17 @@ rtc:
   # # are disabled and clients don't ACK opened peer connections, only reliable, ordered delivery
   # # will be available.
   # strict_acks: true
+  # #
+  # # Settings for improving throughput in high stress forwarding load case.
+  # #
+  # # when set, Livekit will use UDP mux to listen on udp ports (up to runtime.NumCPU()) to
+  # # balance traffic between those ports, also superior to udp_port
```

Comment:
Can we make this transparent & backwards compatible? i.e. if `udp_port` is a single port, then it keeps existing behavior. If user enters a range, we'll support range of ports with higher efficiency.

Reply:
Then we can make the following changes

```
# # when set, LiveKit will attempt to use a UDP mux so all UDP traffic goes through
# # listed port(s). To maximize system performance, we recommend using a range of ports 
# # greater or equal to the number of vCPUs on the machine.
# # port_range_start & end must not be set for this config to take effect
# udp_port: 7882-7892
```